ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/CaseMapperTest.java

```diff
@@ -157,7 +156,7 @@ public void shouldMapSubmissionIndicatorsFromCCDCaseWithNullIndicators() {
     public void shouldMapSubmissionIndicatorsFromCCDCaseWithPinSuccessIndicators() {
         //given
         CCDCase ccdCase =
-            SampleData.getCCDCitizenCaseWithOperationIndicators(getCCDClaimSubmissionOperationIndicatorsWithPinSuccess);
+            SampleData.getCCDCitizenCaseWithOperationIndicators(CCDClaimSubmissionOperationIndicatorsWithPinSuccess);
```

Comment:
`CCDClaimSubmissionOperationIndicatorsWithPinSuccess` seems not following the naming conventions.

Reply:
👍  will change